src/service/resource/repository.ts

```diff
@@ -11,27 +11,27 @@ import { ProbotOctokit } from 'probot';
 import { Entity } from './entity';
 
 export class Repository extends Entity {
-  private repositoryName: string; // uid
+  private readonly _repositoryName: string; // uid
 
   constructor(orgName: string, repoName: string) {
     super(orgName);
-    this.repositoryName = repoName;
+    this._repositoryName = repoName;
```

Comment:
Yes, we use `_repositoryName` prefix not only to indicate as private use, but also leave room for the TypeScript getter.
So that `public get repositoryName()` can take the non-`_` name.